cf-usb/config-production.yaml

```diff
@@ -4,12 +4,14 @@ src-branch: develop
 ci-repo: https://github.com/SUSE/cf-ci.git
 ci-branch: master
 
-docker-server: ""
 docker-tag: latest
+docker-server: "staging.registry.howdoi.website/"
```

Comment:
That should be in the config file, not here.

Reply:
This is a/the `config` file. Or are you talking `cf-usb.yml`, i.e. the pipeline definition instead ?
If yes, where in that definition ?

Reply:
Oh, so it is. Nevermind then, I got confused, sorry. (Unless we want to stick it in the secrets file)

Reply:
👍

Reply:
Done with commit 345074c906484e2478bf6d1289bd535795c298ff, in testing.

Reply:
So it turns out we never set the correct registry in the (helm) configs, so we can't find it.

Reply:
Aaaah, now I understand (on 3rd re-read). The build script has to set the `DOCKER_REPOSITORY` to the one chosen by the config so that the chart has the correct repo baked in. Either that, or subst the correct value into the chart before packaging it into the tarball.

(If we need docker hub during the chart generation)